### PR TITLE
eslint warnings removed from eth1 module

### PR DESCRIPTION
**Pull request by sadiq1971**

### CODE REVIEW

File: src/eth1/wallet.ts [hunk 1]
```diff
@@ -26,7 +25,9 @@ export class Eth1Wallet {
    */
   public async createValidatorDeposit(address: string, amount: BigNumber): Promise<string> {
     let contract = new ethers.Contract(address, defaults.depositContract.abi, this.wallet);
-    //TODO: Implement real deposit arguments according to spec : https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#deposit-arguments
+    //TODO: Implement real deposit arguments according to spec :
```
Comment (GregTheGreek) [inline src/eth1/wallet.ts]: we can remove this

**Review by GregTheGreek — CHANGES_REQUESTED** [review]

File: src/eth1/wallet.ts [hunk 2]
```diff
@@ -26,7 +25,9 @@ export class Eth1Wallet {
...
+    // eslint-disable-next-line max-len
```
Comment (GregTheGreek) [inline src/eth1/wallet.ts]: and this

**Review by GregTheGreek — CHANGES_REQUESTED** [review]

File: src/eth1/defaults.ts [hunk 3]
```diff
@@ -11,8 +11,223 @@ export default {
...
+
```
Comment (wemeetagain) [inline src/eth1/defaults.ts]: We should probably just keep the abi and bytecode on one line. Reason being, this is copy-pasted from the deposit contract repo, we probably want to keep it as similar as possible to the copied input.

**Comment by sadiq1971** [comment]
I have added a new commit updating with master branch and accepting your comments

**Review by wemeetagain — APPROVED** [review]

**Review by GregTheGreek — APPROVED** [review]

**Pull request merged**